Gruntfile.js

```diff
@@ -20,6 +20,8 @@ module.exports = function (grunt) {
          options: {
             verbose: true,
             packageOnly: true,
+            //jsRepository: 'test-files/jsrepository.json',
```

Comment:
These comments are only to test the local files, just wanted to pointed out how to test it. But I can remove these comments if you would prefer :)